src/irc/transport.rs

```diff
@@ -67,6 +67,10 @@ where
                 v.extend_from_slice(line.as_bytes());
                 v.push(b'\n');
             }
+
+            if let Some(ref waker) = inner.waker {
+                waker.wake_by_ref()
```

Comment:
Since the return value isn't used, it seems unusual to leave out the trailing semicolon here.

```suggestion
                waker.wake_by_ref();
```

Reply:
fixed